src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -260,6 +261,21 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
             onDeleteChart: () => {
                 this.props.onDeleteChart(this.props.chartMeta);
             },
+            onComparisonClick: () => {
+                if (this.selectedRowsKeys!.length > 1) {
```

Comment:
we should put a comment here that selectedRowKeys will ALWAYS be gene names.  and also perhaps an explanation of why we do this fork.  i.e when we have a single gene selected we. want to show comparison tab altered/not-altered of results view, whereas for multiple selected genese we want to show group comparison